src/config_test.ts

```diff
@@ -322,6 +322,79 @@ describe('KubeConfig', () => {
         });
     });
 
+    describe('findHome', () => {
+        it('should load from HOME if present', () => {
+            const currentHome = process.env.HOME;
+            process.env.HOME = 'foobar';
+            const dir = join(process.env.HOME, '.kube');
+            const arg = {};
+            arg[dir] = {config: 'data'};
+            mockfs(arg);
+
+            const home = KubeConfig.findHomeDir();
+            expect(home).to.equal(process.env.HOME);
+
+            mockfs.restore();
+            process.env.HOME = currentHome;
```

Comment:
yeah, but the expect uses the env var.... *sigh* I'll move it around...

Reply:
I think if you just capture the constant `foobar` and compare against that then you could expect against that instead of the process environment variable...?